### PR TITLE
Update Dockerfile build + docker run args

### PR DESCRIPTION
**Pull request by nathanleclaire**
Trying to fix https://github.com/workbenchapp/solana-workbench/issues/26 and https://github.com/workbenchapp/solana-workbench/issues/45 as well as consolidate Docker image build from bespoke manual way I've done it previously

### CODE REVIEW

**Comment by nathanleclaire** [comment]
to build the image from the root dir:

```
$ cd docker/solana && docker build -t cryptoworkbench/solana .
```

File: docker/solana/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM rust:slim-buster
+
+RUN rustup toolchain install nightly && rustup default nightly && rustup component add rustfmt
+RUN apt-get update && apt-get install -y git pkg-config libudev-dev make libclang-dev clang
+ENV SOLANA_VERSION v1.8.16
```
Comment (SvenDowideit) [inline docker/solana/Dockerfile:5]: 1.8? booooooo :D

**Comment by nathanleclaire** [comment]
This still doesn't work with `solana program deploy` from your local Mac unfortunately but I wanted to get it up

**Review by SvenDowideit — APPROVED** [review]
I see this as a major improvement that we can build on for the next release

so LGTM

**Pull request merged**